ProjFS.Mac/PrjFSKext/PrjFSKext/VnodeUtilities.cpp

```diff
@@ -12,7 +13,11 @@ FsidInode Vnode_GetFsidAndInode(vnode_t vnode, vfs_context_t context)
     // TODO: check this is correct for hardlinked files
     VATTR_WANTED(&attrs, va_fileid);
 
-    vnode_getattr(vnode, &attrs, context);
+    int errno = vnode_getattr(vnode, &attrs, context);
+    if (0 != errno)
+    {
+        KextLog_FileNote(vnode, "Vnode_GetFsidAndInode: vnode_getattr failed with errno %d", errno);
```

Comment:
How bad is it if we hit this error?  I am wondering if this should be logged at a higher severity (e.g. `Info` or `Error`)